cli/neb.py

```diff
@@ -67,9 +67,12 @@ def get(cls, *, help=False, verbose=False, env=None, col_id=None, col_version=No
                 # because neb refuses to use a dir that already exists
                 neb_dir = join(temp_dir, "neb")
 
-                options = ["--output-dir", neb_dir]
+                options = ["--output-dir", neb_dir, "--insecure"]
```

Comment:
While I think it's useful to be able to run these tests against a local VM, I'm wondering if we want to include `--insecure` all of the time?

One alternative approach to consider could be making it configurable similar to [`neb_env`](https://github.com/openstax/cnx-automation/blob/f7ece379b32335ae90d05dd6d937713d83ffb268/setup.cfg#L5) (e.g. something like `neb_insecure`) and pipe through.

Reply:
If this is a true regression test, wouldn't it be hard to run it against anything _other_ than a VM? Running it against something live and with the potential to change in-between runs seems futile for the purposes of testing for regressions. Maybe my understanding is different since I haven't visited this work in a while. @c0ldcalamari - how are these tests expected to be run?

Reply:
My understanding (which may be incorrect) is that these existing neb tests are run against QA and (perhaps) other environments. I'm not sure about the periodicity or if that's automated somewhere, but running them against a VM with self-signed certs is something new. Not checking the certs isn't going to necessarily hurt those tests, but similar to how we changed `neb` to ignore cert checking only when we know we need that behavior seems like the right way to go versus always ignoring them, no? 🤷‍♂️